flink-mesos/src/test/java/org/apache/flink/mesos/runtime/clusterframework/MesosTaskManagerParametersTest.java

```diff
@@ -233,9 +237,41 @@ public void testConfigCpuCores() {
 		assertThat(mesosTaskManagerParameters.cpus(), is(1.5));
 	}
 
+	@Test
+	public void testUnifiedTotalProcessMemoryConfiguration() {
+		assertTotalProcessMemory(MesosTaskManagerParameters.create(getConfiguration()));
+	}
+
+	@Test
+	public void testLegacyMesosSpecificTotalProcessMemoryConfiguration() {
+		Configuration config = getConfiguration();
+		config.setInteger(MESOS_RM_TASKS_MEMORY_MB, TOTAL_PROCESS_MEMORY_MB);
+		assertTotalProcessMemory(MesosTaskManagerParameters.create(config));
+	}
```

Comment:
How is this test case different from `testUnifiedAndLegacyMesosSpecificTotalProcessMemoryConfigMatchIsOk`? Shouldn't we clear the `TOTAL_PROCESS_MEMORY` config option in `config`?

Reply:
True, forgot to change it to `Configuration config = new Configuration();`